src/interfaces/IPressure.cpp

```diff
@@ -48,29 +40,30 @@ void IPressure::divergence(Field *out, const Field *in_x, const Field *in_y, con
     auto bsize_i = boundary->getSize_innerList();
     auto bsize_b = boundary->getSize_boundaryList();
 
-#pragma acc data present(d_out[:size], d_inx[:size], d_iny[:size], d_inz[:size], d_iList[:bsize_i], d_bList[:bsize_b])
+#pragma acc data present(d_iList[:bsize_i], d_bList[:bsize_b])
+#pragma acc data present(out, inx, iny, inz)
```

Comment:
intentional in two lines? because in the other cases there were one line

Reply:
just for better readability. I can can undo it for constancy.

Reply:
no, it's fine. I just wanted to make sure that it is intentional